src/main/java/homework_1/HomeWork1.java

```diff
@@ -0,0 +1,17 @@
+package homework_1;
+
+public class HomeWork1 {
+
+    private static final String RED = (char) 27 + "[31m";
```

Comment:
Why do you need to use sum here? You could use initial string

Reply:
Do you mean using sum in System.out? I used it to make it clear what those inserts are for.